templates/data/opensearch/what-is-opensearch.html

```diff
@@ -0,0 +1,582 @@
+                                <li class="p-list__item is-ticked">
+                                    Same 10 years of security maintenance covering critical and high severity CVEs in the image
+                                </li>
+                            </ul>
+                        </div>
+                    </div>
+                </div>
+                <div class="row">
+                    <hr class="is-muted" />
+                    <div class="col-3">
+                        <h3 class="p-heading--5">OpenSearch consultancy and support</h3>
+                    </div>
+                    <div class="col-6">
+                        <h3 class="p-heading--5">Advanced professional services for OpenSearch, when you need them</h3>
+                        <p>
+                            Get help designing, planning and building and even operating a hyper automated production OpenSearch service that perfectly fits your needs, with Canonical’s expert services.
+                        </p>
+                        <hr class="is-muted" />
+                        <div class="p-section--shallow">
+                            <ul class="p-list--divided">
+                                <li class="p-list__item is-ticked">
+                                    Help with design and build of both production and non-production OpenSearch environments with Charmed OpenSearch
+                                </li>
+                                <li class="p-list__item is-ticked">
+                                    Managed services for OpenSearch in your cloud tenancy or data centre, backed by an SLA
+                                </li>
+                                <li class="p-list__item is-ticked">
+                                    Firefighting support with a OpenSearch operations expert, who works alongside your team when crisis hits
+                                </li>
+                            </ul>
+                            <hr class="is-muted" />
+                            <p>
+                                <a href="https://assets.ubuntu.com/v1/4220f815-Data%20Solutions%20Advisory%20version%20_updated%20-%20v3-2.pdf">Access the datasheet
+                                &nbsp;&rsaquo;</a>
+                            </p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section class="p-section">
+        <div class="row--50-50">
+            <hr />
+            <div class="col">
+                <h2>Installing Charmed OpenSearch</h2>
+            </div>
+            <div class="col">
+                <p>
+                    Charmed OpenSearch  is operated via Juju, an open source orchestration engine for software operators that makes OpenSearch easy to operate. After <a href="https://charmhub.io/opensearch/docs/t-setup-environment">setting-up the environment</a>, users can deploy Charmed OpenSearch with one command:
```

Comment:
This link 404s

Reply:
This page is still in development and hence this branch merges to feature branch.
Once all the pages are made this will be merged to main.